org.scala-ide.sdt.core/src/org/scalaide/core/compiler/DeclarationPrinter.scala

```diff
@@ -39,7 +39,7 @@ abstract class DeclarationPrinter extends HasLogger {
    *
    *  @note Should be run on the presentation compiler thread (inside askOption).
    */
-  def defString(sym: Symbol, flagMask: Long = Flags.ExplicitFlags, showKind: Boolean = true)(seenAs: Type = sym.rawInfo): String = {
+  def defString(sym: Symbol, flagMask: Long = Flags.ExplicitFlags, showKind: Boolean = true, showAccessor: Boolean = true)(seenAs: Type = sym.rawInfo): String = {
```

Comment:
I'm just thinking but instead of `showAccessor` flag maybe inner function could be used like:
`def showAccessor = flagMask != 0L && (flagMask & AccessFlags) == 0  ?